client/extensions/woocommerce/state/selectors/promotions.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteWithFallback } from '../sites/selectors';
+
+export function getPromotions( rootState, siteId = getSelectedSiteWithFallback( rootState ) ) {
+	const { promotions } = rootState.extensions.woocommerce.sites[ siteId ];
+	return promotions.promotions;
```

Comment:
Should we return an empty array here if no promotions are found? We do this for our other similar selectors.

Reply:
FWIW, use have used `lodash` `get` for things like this (both getting the item from state and returning an appropriate default)

Reply:
In this case, I want the `undefined` return when the promotions haven't yet been loaded, so I can reflect that with placeholders in the UI.

Reply:
> FWIW, use have used lodash get...

Yeah, I don't always bring it in until I have more need for it. But I think I add it in a later PR anyway...